guides/common/modules/ref_storage-requirements.adoc

```diff
@@ -26,6 +26,8 @@ endif::[]
 ifdef::katello,satellite,orcharhino[]
 |/var/lib/pulp |1 MB |300 GB
 
+|/var/lib/containers |10 GB only if {Insights} is enabled |15 GB
```

Comment:
```suggestion
|/var/lib/containers |10 GB if using {Insights} |15 GB
```
Will this make it fit? :) Let's try.

Reply:
I would argue that this would only apply if we're talking about IoP and not regular Insights.

Reply:
Correct, isn't that what the macro resolves to?